File: JS/classes/uber-classes/uberVan.js
const SpecialUberCar = require("./specialUberCar").SpecialUberCar;

class UberVan extends SpecialUberCar{
    constructor(license,driver,typeCarAccepted,seatsMaterial){
        super(license,driver,typeCarAccepted,seatsMaterial);
    }
}

exports.UberVan = UberVan;

File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends SpecialUberCar{

    public UberVan(String license, Account driver,Map<String, Map<String,Integer>> typeCarAccepted,ArrayList<String> seatsMaterial){
        super(license, driver, typeCarAccepted, seatsMaterial);
    }
}